animationHelpers.js

```diff
@@ -81,9 +73,9 @@ function animate(duration = 1000) {
         [xTransVal, yTransVal, zTransVal] = interpolatedTranslation[currentStep];
 
         renderOnce();
-
+        vertexArray = getFrameData();
         currentStep++;
```

Comment:
!Wrap right hand side of all vertexArray assignments inside getVertices() function.